networking/hardware_networks/uninstalling-sriov-operator.adoc

```diff
@@ -0,0 +1,50 @@
+[id="uninstalling-sriov-operator"]
+= Uninstalling the SR-IOV Network Operator
+include::modules/common-attributes.adoc[]
+:context: uninstalling-sr-iov-operator
+
+toc::[]
+
+To uninstall the SR-IOV Network Operator, you must first delete any running SR-IOV workloads. You must also delete the webhooks after uninstalling the operator.
+
+.Prerequisites
+
+* You must have access to an {product-title} cluster using an account with `cluster-admin` permissions.
+* You have the SR-IOV Network Operator installed.
+
+.Procedure
+
+. Delete all SR-IOV custom resources (CRs).
```

Comment:
Could we enumarate the resources to remove?

oc delete sriovnetwork  -n openshift-sriov-network-operator --all
oc delete sriovnetworknodepolicy  -n openshift-sriov-network-operator --all
oc delete sriovibnetwork  -n openshift-sriov-network-operator --all

Maybe also add a note:
NOTE: it is imporant to delete these resource before removing the SRIOV operator. Failing to remove them will result in stray configuration item being left in the cluster.

Failing to remove networks will cause stray network attachment-defintions to stay, so it's important to delete this before removing the operator.

Reply:
Good suggestion on enumerating the resources to remove. Thanks for sharing the commands!

Regarding the note, I think it is unnecessary. We are describing the removal step-by-step, customers have no reasons to believe they can skip any step and the impact that would cause to the cluster.